system/Helpers/filesystem_helper.php

```diff
@@ -103,7 +103,7 @@ function directory_mirror(string $originDir, string $targetDir, bool $overwrite
 
 		$dirLen = strlen($originDir);
 
-		/** @var SplFileInfo $file */
+		// @var SplFileInfo $file
```

Comment:
I don't think it is correct to change to simple comment. IDEs will not be able to parse the docblock